library/src/main/java/com/alexfu/countdownview/CountDownView.java

```diff
@@ -34,6 +34,7 @@
     private long startDuration;
     private long currentDuration;
     private boolean timerRunning;
+    private OnCountListener listener;
```

Comment:
Since `listener` can be nullable, please add a `@Nullable` annotation to this field.